resources/web/docs_js/index.js

```diff
@@ -379,6 +405,30 @@ $(function() {
     }, 150);
   }
 
+  window.dataLayer = window.dataLayer || [];
+
+  let titleParams = document.title.split('|')
+
+  let pageViewData = {
+    'event': 'page_view',
+    'pagePath': window.location.pathname,
```

Comment:
Should these be uppercase `PagePath` or lower case `pagePath`. I'm seeing differences between the specification and the examples on www.elastic.co.